### ISSUE TITLE
Provide a way to not install prometheus if already installed

### ISSUE THREAD

**Issue by smruthi2187** [fission/fission#915]
If prometheus is installed on the cluster, fission should use the service of the prometheus server and should not have to re-install prometheus.

**Comment by soamvasani** [comment]
Also, even if it's not installed, we should add a flag that disables prometheus installation.

**Comment by smruthi2187** [comment]
In that case, the canary feature cannot be used by the users.

**Comment by soamvasani** [comment]
Yeah, fair enough.  We should mention that in the installation docs once we add the flag.
